<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bit_them_all</class>
 <widget class="QMainWindow" name="bit_them_all">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="refreshButton">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="Map">
       <attribute name="title">
        <string>Map</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Fleets">
       <attribute name="title">
        <string>Fleets</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTreeWidget" name="fleetTable">
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Z</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Content</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ressources</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Planets">
       <attribute name="title">
        <string>Planets</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTreeWidget" name="planetTable">
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Z</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ressource</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Reports">
       <attribute name="title">
        <string>Reports</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeWidget" name="reportTreeWidget">
           <attribute name="headerDefaultSectionSize">
            <number>100</number>
           </attribute>
           <column>
            <property name="text">
             <string>Time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Comment</string>
            </property>
           </column>
          </widget>
          <widget class="QTableView" name="tableView"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Code">
       <attribute name="title">
        <string>Code</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="codeTab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="Fleet">
           <attribute name="title">
            <string>Fleet</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
              <item>
               <widget class="QTextEdit" name="fleetCodeEdit">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="lineWrapMode">
                 <enum>QTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidget">
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>60</number>
                </attribute>
                <column>
                 <property name="text">
                  <string>Version</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Tag</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="resetFleetCodeButton">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saveFleetCodeButton">
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="Planet">
           <attribute name="title">
            <string>Planet</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,1">
              <item>
               <widget class="QTextEdit" name="planetCodeEdit">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="lineWrapMode">
                 <enum>QTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidget_4">
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>60</number>
                </attribute>
                <column>
                 <property name="text">
                  <string>Version</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Tag</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="resetPlanetCodeButton">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="savePlanetCodeButton">
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionUnlog"/>
    <addaction name="actionLog"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEngine">
    <property name="title">
     <string>Engine</string>
    </property>
    <addaction name="actionNew_2"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuEngine"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUnlog">
   <property name="text">
    <string>log out</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log in</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>bit_them_all</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
